pkg/ui/app/containers/nodeGraphs.tsx

```diff
@@ -446,15 +521,26 @@ class NodeGraphs extends React.Component<NodeGraphsProps, {}> {
           <SummaryStat title="Total Nodes" value={this.props.nodeCount} />
           <SummaryStat title="Capacity Used" value={capacityPercent}
                        format={(v) => `${d3.format(".2f")(v)}%`}
-                       tooltip={`You are using ${Bytes(capacityUsed)} of ${Bytes(capacityTotal)} 
+                       tooltip={`You are using ${Bytes(capacityUsed)} of ${Bytes(capacityTotal)}
                        storage capacity across all nodes.`} />
+          <SummaryStat title="Unavailable ranges" value={this.props.unavailableRanges} />
+          <SummaryStat title="Queries per second" value={this.props.queriesPerSecond} />
+          <SummaryStat title="P50 latency" value={this.props.p50Latency} format={(n) => d3.format(".1f")(NanoToMilli(n)) + " ms"} />
```

Comment:
So let me ask you here; why did you not wrap these two SummaryStats in their own MetricDataProvider objects?  Creating something like this:

```typescript
<MetricDataProvider id="node.summary.1">
  <MetricSummaryStat title="Queries per second">
     <Metric name="cr.node.sql.query.count" NonNegativeRate />
  </MetricSummaryStat>
</MetricDataProvider>
```

MetricDataProvider is already programmed to do this for arbitrary components, not just graphs.  Look at the class `TextGraph` in `graphs.tsx` - MetricsDataProvider scours its children for `Metric` components, queries the data requested, and then attached the results to the outermost object via a `data` property.

So, you would just need to program `MetricSummaryStat`, which accepts metrics data via its `data` property and displays the last point in a SummaryStat.

As we discussed, the other thing you would need to do is create a version of MetricsDataProvider which is not `connect`ed to the current global time window. This is relatively simple; MetricsDataProvider can work with any time window, it is joined to the global time window via `connect`; we can export two different `connect`ed versions of this class, one which is connected only to the data and the other (existing one) which is also connected to the global time window.

This lets you implement this without re-inventing the wheel on querying metrics.